src/cryptonote_basic/cryptonote_format_utils.cpp

```diff
@@ -1012,6 +1012,21 @@ namespace cryptonote
     return true;
   }
   //---------------------------------------------------------------
+  bool tx_outs_checked_for_torsion(const transaction& tx)
```

Comment:
I would move this to `tx_verification_utils` since it's a function which simply takes transaction info, performs query, and returns `true`/`false.

Reply:
I agree in spirit. The problem is that would make `blockchain_db_utils` -> `cryptonote_core` -> `blockchain_db` -> `blockchain_db_utils` so makes a circular dep.

Could solve for this by moving `get_outs_by_last_locked_block` into tx verification utils as well, then modify `add_block` to take in that result. Probably the way to go.

Would prefer to do this in a separate PR, since this function's placement is a minor point for this PR, and requires more changes outside of the scope of this PR.